packages/components/managed-data-table/src/managed-data-table.story.js

```diff
@@ -0,0 +1,508 @@
+            isChecked={getIsRowSelected(row.id)}
+            onChange={() => toggleRow(row.id)}
+          />
+        ),
+        disableResizing: true,
+      },
+      ...visibleColumns,
+    ];
+
+    return (
+      <React.Fragment>
+        <ManagedDataTable
+          rows={withRowSelection ? rowsWithSelection : rows}
+          columns={withRowSelection ? columnsWithSelect : visibleColumns}
+          onRowClick={
+            onRowClick
+              ? (item, index) => alert(`Row click: Row number ${index}`)
+              : null
+          }
+          maxHeight={number('maxHeight', 0, {
+            range: true,
+            min: 200,
+            max: 500,
+            step: 10,
+          })}
+          maxWidth={number('maxWidth', 0, {
+            range: true,
+            min: 200,
+            max: 800,
+            step: 10,
+          })}
+          cellAlignment={select('cellAlignment', ['left', 'center', 'right'])}
+          isCondensed={isCondensed}
+          isWrappingText={isWrappingText}
+          wrapHeaderLabels={boolean('wrapHeaderLabels', true)}
+          disableHeaderStickiness={boolean('disableHeaderStickiness', false)}
+          disableFooterStickiness={boolean('disableFooterStickiness', false)}
+          sortedBy={sortedBy}
+          onSortChange={onSortChange}
+          sortDirection={sortDirection}
+          footer={footer}
+          useDisplaySettings={boolean('useDisplaySettings', true)}
+          useColumnManager={boolean('useColumnManager', true)}
+          visibleColumnKeys={tableData.visibleColumnKeys}
+          hideableColumns={tableData.columns}
+          onUpdateColumns={(visibleColumnKeys) =>
+            setTableData({
+              ...tableData,
+              visibleColumnKeys,
+            })
+          }
+          onIsCondensedChange={setIsCondensed}
+          onIsWrappingTextChange={setIsWrappingText}
+          topBar={topBar}
+        />
+        <br />
+        <hr />
+        <div>
+          <h3>{'Configure Column Settings'}</h3>
+          <p>
+            Note: the <code>width</code> input accepts the same values as the
+            ones specified for{' '}
+            <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns">
+              grid-template-columns
+            </a>
+            . Try values such as <code>minmax(200px, 400px)</code>, combinations
+            of <code>1fr</code> and <code>2fr</code>, or simply fixed values
+            like <code>100px</code>. The default is <code>auto</code>.
+          </p>
+          <div style={{ display: 'flex' }}>
+            {tableData.columns.map((col, colIndex) => (
+              <ColumnConfigForm
+                key={col + colIndex}
+                updateColumn={(column) => handleUpdateColumn(column, colIndex)}
+                column={col}
+              />
+            ))}
+          </div>
+          <hr />
+          <h4>{'Select an input type to test on the second column'}</h4>
```

Comment:
Perhaps we could replace this part of the example with a customRenderer with a special truncation style!